### DISCUSSION TITLE
Telegram Yardım Botu

### DISCUSSION THREAD

**Discussion by mtkucuk** [acikkaynak/deprem-yardim-projesi#40] [Ideas]
Merhabalar, 

Faydalı olur mu bilmiyorum ama telegram bot oluşturup ordan gelebilecek talepleri ilgili kişiler veya yetkililerle yönlendirme ile ilgili çalışma olur mu?

Aynı zamanda minik bir site üzerinden bunlar yayınlanabilir.

Bunla ilgili bir çalışmaya başladım.

Eğer uygunluğu konusunda doğru bir yönlendirme yaparsanız sevinirim.

Repo : https://github.com/mtkucuk/telegram-afet-yardim-bot

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle fikriniz için teşekkürler. Talebinizi [bu](https://github.com/acikkaynak/afet-yardim-telegram-bot) repo 'da issues olarak açabilirsiniz.

**Comment by mtkucuk** [comment]
Merhabalar,

İlgili talebi issue olarak açtım.

Faydalıysa ve gerekli yönlendirmeleri yaparsanız sevinirim.

Herkese kolay gelsin.
